src/Authentication/Services/SystemUserService.cs

```diff
@@ -300,5 +304,14 @@ public async Task<Result<SystemUser>> CreateAndDelegateSystemUser(string partyId
         {
             return await _repository.GetSystemUserByExternalRequestId(externalRequestId);
         }
+
+        /// <inheritdoc/>
+        public async Task<Result<Page<SystemUserRegisterDTO, long>>> GetAllSystemUsers(long continueFrom, CancellationToken cancellationToken)
+        {
+            List<SystemUserRegisterDTO>? theList = await _repository.GetAllSystemUsers(continueFrom, _paginationSize, cancellationToken);
+            theList ??= [];
+
+            return Page.Create(theList, _paginationSize, static theList => theList.SequenceNo);
```

Comment:
Calling the argument `theList` here probably doesn't make sense. It's not a list, it's an item.